--- webkitgtk-2.24.1/Source/WebKit/PlatformGTK.cmake.orig	2019-04-08 11:30:53.000000000 +0200
+++ webkitgtk-2.24.1/Source/WebKit/PlatformGTK.cmake	2019-04-22 09:07:32.025569199 +0200
@@ -855,6 +855,11 @@
 
     ADD_TYPELIB(${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.typelib)
     ADD_TYPELIB(${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.typelib)
+
+    # ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir is created from different directory
+    # so cmake needs top-level target dependency
+    # note: NAME_WE cuts all extensions, so WebKit2-${WEBKITGTK_API_VERSION}.gir becomes WebKit2-4 not WebKit2-4.0
+    add_dependencies(WebKit2-4-gir JavaScriptCore-4-gir)
 endif ()
 
 install(TARGETS webkit2gtkinjectedbundle
