--- webkitgtk-2.32.0/Source/WebKit/PluginProcess/PluginControllerProxy.h.orig	2021-02-26 10:57:15.000000000 +0100
+++ webkitgtk-2.32.0/Source/WebKit/PluginProcess/PluginControllerProxy.h	2021-03-28 11:40:29.813445533 +0200
@@ -110,11 +110,6 @@ private:
     bool getAuthenticationInfo(const WebCore::ProtectionSpace&, String& username, String& password) override;
     void protectPluginFromDestruction() override;
     void unprotectPluginFromDestruction() override;
-#if PLATFORM(X11)
-    uint64_t createPluginContainer() override;
-    void windowedPluginGeometryDidChange(const WebCore::IntRect& frameRect, const WebCore::IntRect& clipRect, uint64_t windowID) override;
-    void windowedPluginVisibilityDidChange(bool isVisible, uint64_t windowID) override;
-#endif
     
     // Message handlers.
     void frameDidFinishLoading(uint64_t requestID);
--- webkitgtk-2.32.0/Source/WebKit/PluginProcess/PluginControllerProxy.cpp.orig	2021-02-26 10:57:15.000000000 +0100
+++ webkitgtk-2.32.0/Source/WebKit/PluginProcess/PluginControllerProxy.cpp	2021-03-28 11:41:02.849933225 +0200
@@ -651,25 +651,6 @@ void PluginControllerProxy::getFormValue
     completionHandler(returnValue, WTFMove(formValue));
 }
 
-#if PLATFORM(X11)
-uint64_t PluginControllerProxy::createPluginContainer()
-{
-    uint64_t windowID = 0;
-    m_connection->connection()->sendSync(Messages::PluginProxy::CreatePluginContainer(), Messages::PluginProxy::CreatePluginContainer::Reply(windowID), m_pluginInstanceID);
-    return windowID;
-}
-
-void PluginControllerProxy::windowedPluginGeometryDidChange(const IntRect& frameRect, const IntRect& clipRect, uint64_t windowID)
-{
-    m_connection->connection()->send(Messages::PluginProxy::WindowedPluginGeometryDidChange(frameRect, clipRect, windowID), m_pluginInstanceID);
-}
-
-void PluginControllerProxy::windowedPluginVisibilityDidChange(bool isVisible, uint64_t windowID)
-{
-    m_connection->connection()->send(Messages::PluginProxy::WindowedPluginVisibilityDidChange(isVisible, windowID), m_pluginInstanceID);
-}
-#endif
-
 } // namespace WebKit
 
 #endif // ENABLE(NETSCAPE_PLUGIN_API)
