--- webkitgtk-2.50.4/Source/cmake/WebKitMacros.cmake.orig	2025-08-08 11:17:56.304456000 +0200
+++ webkitgtk-2.50.4/Source/cmake/WebKitMacros.cmake	2026-01-23 12:48:45.426052097 +0100
@@ -16,6 +16,8 @@
     # Windows needs a larger bundle size because that helps keep WebCore.lib's size below the 4GB maximum in debug builds.
     if (MSVC AND ${_framework} STREQUAL "WebCore" AND ${_framework}_LIBRARY_TYPE STREQUAL "STATIC")
         list(APPEND gusb_args --max-bundle-size 16)
+    elseif (UNIFIED_BUILD_MAX_BUNDLE_SIZE)
+        list(APPEND gusb_args --max-bundle-size ${UNIFIED_BUILD_MAX_BUNDLE_SIZE})
     endif ()
 
     if (ENABLE_UNIFIED_BUILDS)
--- webkitgtk-2.50.6/Source/WebKit/WebProcess/InjectedBundle/API/gtk/DOM/WebKitDOMEvent.cpp.orig	2026-06-13 18:16:39.655384811 +0200
+++ webkitgtk-2.50.6/Source/WebKit/WebProcess/InjectedBundle/API/gtk/DOM/WebKitDOMEvent.cpp	2026-06-13 18:16:58.275283938 +0200
@@ -29,6 +29,7 @@
 #include "WebKitDOMEventTargetPrivate.h"
 #include "WebKitDOMPrivate.h"
 #include "ConvertToUTF8String.h"
+#include <WebCore/EventTargetInlines.h>
 #include <wtf/GetPtr.h>
 #include <wtf/RefPtr.h>
 
