From: Daniel Schepler <dschepler@gmail.com>
Subject: Fix FTBFS in x32
Bug-Debian: https://bugs.debian.org/700795
Index: webkitgtk/Source/WTF/wtf/Platform.h
===================================================================
--- webkitgtk.orig/Source/WTF/wtf/Platform.h
+++ webkitgtk/Source/WTF/wtf/Platform.h
@@ -179,7 +179,11 @@
 /* CPU(X86_64) - AMD64 / Intel64 / x86_64 64-bit */
 #if   defined(__x86_64__) \
     || defined(_M_X64)
+#ifdef __ILP32__
+#define WTF_CPU_X86_64_32 1
+#else
 #define WTF_CPU_X86_64 1
+#endif
 #define WTF_CPU_X86_SSE2 1
 #define WTF_CPU_KNOWN 1
 #endif
--- webkitgtk-2.24.1/Source/bmalloc/bmalloc/Gigacage.h.orig	2019-04-20 18:12:00.719079717 +0200
+++ webkitgtk-2.24.1/Source/bmalloc/bmalloc/Gigacage.h	2019-04-20 18:17:04.960764832 +0200
@@ -35,7 +35,7 @@
 #include <inttypes.h>
 
 #if ((BOS(DARWIN) || BOS(LINUX)) && \
-(BCPU(X86_64) || (BCPU(ARM64) && !defined(__ILP32__) && (!BPLATFORM(IOS_FAMILY) || BPLATFORM(IOS)))))
+((BCPU(X86_64) && !defined(__ILP32__)) || (BCPU(ARM64) && !defined(__ILP32__) && (!BPLATFORM(IOS_FAMILY) || BPLATFORM(IOS)))))
 #define GIGACAGE_ENABLED 1
 #else
 #define GIGACAGE_ENABLED 0
